ConwaysGameOfLife.Tests/UniverseShould.cs

```diff
@@ -134,7 +134,7 @@ public void WhenTwoUniversesInstantiated_BeEqual()
         }
 
         [Fact]
-        public void WhenTwoUniversesInstantiated_NotBeEqual()
+        public void WhenUniversesOfDifferentDimensionsInstantiated_NotBeEqual()

```

Comment:
the test is not doing what the name is suggesting.

Reply:
could be a sign that the helper method isn't doing as it should i.e.
- shouldn't throw exceptions
- should always return true or false